pkg/constants/env_variables.go

```diff
@@ -9,5 +9,7 @@ const (
 )
 
 const (
-	AllowedRegistries = "ALLOWED_REGISTRY"
+	AllowedRegistries                             = "ALLOWED_REGISTRY"
+	SkipPluginMetadataSignatureVerification       = "SKIP_PLUGIN_METADATA_SIGNATURE_VERIFY"
+	CustomPublicKeyPathForPluginMetadataSignature = "PLUGIN_METADATA_SIGNATURE_CUSTOM_PUBLIC_KEY_PATH"
```

Comment:
Considering we are using `ADDITIONAL_PLUGIN_DISCOVERY_IMAGES_TEST_ONLY` for another variable and here we are verifying the discovery image how about using something like that
- `SKIP_PLUGIN_DISCOVERY_IMAGE_SIGNATURE_VERIFICATION`
- `PLUGIN_DISCOVERY_IMAGE_SIGNATURE_PUBLIC_KEY_PATH`

WDYT about this suggestion? @prkalle @marckhouzam @vuil

Reply:
Thanks @anujc25 . I like your suggestion, but in your suggestion for `PLUGIN_DISCOVERY_IMAGE_SIGNATURE_PUBLIC_KEY_PATH` you want to remove `CUSTOM` word too, right?

Reply:
Yeah. So, my understanding is we will be using this for future key-rotation use cases. And I felt the `CUSTOM` word is not needed in that case.

Reply:
I like the names suggested by @anujc25 , I think they'll be easier to understand.

Reply:
Updated the variables as below(with TANZU_CLI prefix to the environment variables)
`TANZU_CLI_SKIP_PLUGIN_DISCOVERY_IMAGE_SIGNATURE_VERIFICATION`
`TANZU_CLI_PLUGIN_DISCOVERY_IMAGE_SIGNATURE_PUBLIC_KEY_PATH`